experimental/report/testdata/suggestions.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020-2024 Buf Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# buf.compiler.v1.Report
+
+files:
+  - path: "foo.proto"
+    text: |
+      syntax = "proto3";
+
+      package abc.xyz;
+
+      service Foo {
+        rpc Get(GetRequest) returns GetResponse;
+        rpc Put(PutRequest) returns (PutResponse) [foo = bar];
+      }
+
+diagnostics:
+  - message: 'let protocompile pick a syntax for you'
```

Comment:
This was confusing to read at first. I think this would be very helpful to add comments in here that none of these are actual suggestions that protocompile will make but instead just examples to test the rendering of suggestions.